product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Security"
```

Comment:
Nit: should we remove "compliance" from the page title here, too?

Reply:
@skrivov-edb can you proceed with this?